src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -332,6 +337,21 @@ const formConfig = {
             newDisabilities: state.form?.data?.newDisabilities || [],
           }),
         },
+        addDisabilitiesRevised: {
+          title: 'Add a new disability',
+          path: DISABILITY_SHARED_CONFIG.addDisabilities.path,
```

Comment:
I wonder if adding two pages with the same path could cause issues, even with the depends clauses